cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,25 @@
+import socket
+import time
+import os
+
+from django.core.management import BaseCommand
+
+
+class Command(BaseCommand):
+    help = "Waits for database to be ready for connection"  # noqa: VNE003
+
+    def handle(self, *args):
+        port = int(os.environ["POSTGRES_PORT"])  # 5432
+
+        db_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+
+        print("Waiting for DB connection...")
+
+        while True:
+            try:
+                db_socket.connect(("db", port))
+                print("DB connection established.")
+                db_socket.close()
+                break
+            except socket.error:
+                time.sleep(0.1)
```

Comment:
Consider adding a maximum number of retries or a timeout for the connection attempts. As it stands, this loop could potentially run forever if the database never becomes available. You could, for instance, add a counter and break the loop if a certain number of attempts have been made, or check the elapsed time since the first attempt and break the loop if it exceeds a certain threshold.